image-loader/app/model/ImageUpload.scala

```diff
@@ -119,4 +160,13 @@ case object ImageUpload {
     )
   }
 
+  private def isPng24(mimeType: Option[String], fileMetadata: FileMetadata): Boolean =
+    mimeType match {
+      case Some("image/png") => {
+        val colourType = fileMetadata.colourModelInformation.get("colorType").getOrElse("")
```

Comment:
apologies as you've already changed this!

Is the use of empty string necessary? As it looks like we're using empty string for `None` which feels odd. Can it be replaced with a pattern match? Though I may be misunderstanding the code!

Something like:

```scala
mimeType match {
  case Some("image/png") -> {
    fileMetadata.colourModelInformation.get("colourType") match {
      case Some("True Colour") => true
      case Some("True Colour with Alpha") => true
      case _ => false
    }
  }
  case _ => false
}
```